### ISSUE TITLE
[Bug Report][3.7.3] Aria-Controls Attribute Not Present on Expanded V-Autocomplete

### ISSUE THREAD

**Issue by gracecicc** [vuetifyjs/vuetify#20843]
### Environment
**Vuetify Version:** 3.7.3
**Vue Version:** 3.4.34
**Browsers:** Chrome 131.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Go to reproduction link
Open the autocomplete.
Using arrow keys, focus one of the options in the autocomplete menu. This way the menu will stay open when you run the accessibility scan.
Open axe accessibility tools and run scan of page.
See error: 'Required Aria Attributes Must be Provided'. Error description shows that the active v-autocomplete is missing the aria-controls attribute.

### Expected Behavior
Aria-controls attribute should be updated when autocomplete becomes active.

### Actual Behavior
Aria-controls attribute is not present at all.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpdj8FuwyAQRH9ly4VWcuq75UTqrX/QQ5wDcjYN0mIQLK4qy//eNUiVkwti4O3MznlRKY7tRwjvc0bVqZ7RBTKMp2EC6OeDCaFcixj9xMZOGE/lK7MfveDICPPB+SvScVAufQ8KOitOqcokupWRdu9Q/OWlBvTtLlhkGqMNDAk513zrgo8MC0S8wQq36B1o2VlvOID4JgbJguNGvOpPJPLw5SNdX/TbA5KEOddOC7Blwg70z/1XNzAbypuSTiahhrV55u5IYQe6f+iylahry8JqbaZMVI7LH1MVdug=)

**Comment by ikushum** [comment]
Fixed already in this PR -https://github.com/vuetifyjs/vuetify/pull/22025
